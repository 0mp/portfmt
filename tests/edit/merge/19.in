USE_GL=		gl glu

LLD_UNSAFE=	yes

MAKE_ARGS=	CC="${CC}" CXX="${CXX}"
