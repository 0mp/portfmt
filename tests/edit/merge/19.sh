${PORTEDIT} merge -e 'LLD_UNSAFE!=' 19.in | diff -L 19.expected -L 19.actual -u 19.expected -
