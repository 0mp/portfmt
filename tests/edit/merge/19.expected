USE_GL=		gl glu

MAKE_ARGS=	CC="${CC}" CXX="${CXX}"
